### ISSUE TITLE
Show Status/Ping for the proxy that user selects to connect from proxy channels

### ISSUE THREAD

**Issue by SaeedZhiany** [telegramdesktop/tdesktop#27856]
### Is your feature request related to a problem?

no

### Describe the solution you'd like

| Telegram desktop | Telegram android |
|---------------------|--------------------|
|<img width="296" alt="image" src="https://github.com/telegramdesktop/tdesktop/assets/13766432/e90870ad-8983-4c60-b287-f1431a44683e">|![photo_2024-05-05_14-20-21](https://github.com/telegramdesktop/tdesktop/assets/13766432/05a926e7-11aa-4820-b1f1-cf3504ffdfd7)|

### Describe alternatives you've considered

In Telegram for Android, when users attempt to connect to a new proxy from proxy channels, a pop-up displays the proxy’s information. One piece of information is the proxy’s Status and Ping, which indicates whether the proxy is still active. If it’s not, users should simply avoid trying to add or connect to the proxy.

Telegram Desktop does not have this feature, and it would be beneficial to add it.

**Comment by ilya-fedin** [comment]
This is already present in proxy settings

**Comment by kr-ilya** [comment]
I'll agree. I think it means the output of information directly to the popup, so that you can easily find out the ping, without adding a proxy and opening the proxy settings. This is a user friendly feature.

**Comment by SaeedZhiany** [comment]
> This is already present in proxy settings

@ilya-fedin, yes, I’m aware of that. However, what I’m suggesting is to display the ping in the pop-up even before adding the proxy to the user’s proxy list, exactly as @kr-ilya mentioned.

In the desktop version, we only have the ‘Server’ and ‘Port’ information in the pop-up. For me, having the ‘Ping’ information is more important than knowing the proxy’s ‘Server’ and ‘Port’ address when choosing a proxy to connect. It helps me avoid adding inactive proxies and keeps my proxy list organized.
